views/chorals.pug

```diff
@@ -30,3 +30,9 @@ block content
           td
             button.btn.btn-danger.delete-choral-btn(data-choral-id=choral.choralId)
               span.glyphicon.glyphicon-trash
+
+    script.
+      $('.choral-row').on("click", function() {
+        document.location = $(this).data('href');
```

Comment:
I think a button is the best choice. Blue button with a graph? To the left of the delete button. If @james-woo or @ecuellog  think this is no good speak now.